### PR TITLE
Create gokulbarath.json

### PR DESCRIPTION
**Pull request by gokul6350**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by gokul6350** [comment]
i guss it will work

**Comment by gokul6350** [comment]
wdwd

File: domains/gokulbarath.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Personal Website",
+    "repo": "https://github.com/gokul6350/gokul6350.github.io",
+    "owner": {
+        "username": "gokul6350",
+        "email": "gokul00060@gmail.com",
+        "twitter": "gokul00060"
+    },
+    "record": {
+        "CNAME": "gokul6350.github.io"
```
Comment (DEV-DIBSTER) [inline domains/gokulbarath.json:10]: Little content provided.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/gokulbarath.json (hunk 1 above)
Comment (gokul6350) [reply domains/gokulbarath.json:10]: what content u need more ?

**Pull request closed without merging**